Add certificate page

EthanThatOneKid opened the pull request:
### At a glance

The certificate page recognizes individual contributors for their contributions to `acmcsuf.com`.

### Background

This surprise is intended to recognize the `acmcsuf.com` contributors for helping cultivate a culture of friendliness while contributing to `acmcsuf.com` (most recently, our Fall 2022 group who worked on v4.0) in our Discord server and during in-person meetings throughout the semesters.

This project is what it is today thanks to the effort of its contributors!

### Features

- Showcase the PRs of a contributor from a GitHub release
  - Like: `https://acmcsuf.com/@$USERNAME?release=$RELEASE` 
- View PRs from a list of [official `acmcsuf.com` releases](https://github.com/EthanThatOneKid/acmcsuf.com/releases)

### Screenshot

|Light theme | Dark theme|
|---|---|
|<img width="431" alt="image" src="https://user-images.githubusercontent.com/31261035/213942941-a7b3109c-1ade-4e36-8066-d612114cf559.png">|<img width="431" alt="image" src="https://user-images.githubusercontent.com/31261035/213943046-221070e5-e94c-438a-9318-d427c23fe277.png">|

EthanThatOneKid committed:
Progress

EthanThatOneKid committed:
Progress 2

EthanThatOneKid committed:
Merge branch 'main' into add/certificates

EthanThatOneKid committed:
Progress 🎆

New Year's Eve progress!

EthanThatOneKid committed:
Progress

EthanThatOneKid committed:
Omit unused code

Added cache logic to `src/routes/(site)/%40[username]/[release].json/+server.ts`.

EthanThatOneKid committed:
Merge branch 'main' into add/certificates

EthanThatOneKid committed:
Fix `/@username` page styles

TODO: Supply profile data with releases data.

EthanThatOneKid committed:
Fix `/@username` page styles

TODO: Supply profile data with releases data.

EthanThatOneKid committed:
Merge branch 'add/certificates' of https://github.com/EthanThatOneKid/acmcsuf.com into add/certificates

EthanThatOneKid committed:
Provide profile data to make the page more specific

EthanThatOneKid committed:
Slight word choice edit

EthanThatOneKid committed:
Use HTML from GitHub bio

EthanThatOneKid committed:
Update page formatting/styling

EthanThatOneKid committed:
Throw early if user is not found

EthanThatOneKid committed:
Pass tests

EthanThatOneKid committed:
Add OpenGraph page description

k4rni commented on vite.config.ts.timestamp-1674429041045.mjs:
```diff
@@ -0,0 +1,8 @@
+// vite.config.ts
```
What is this file for?

EthanThatOneKid replied:
No clue! Resolved by <https://github.com/EthanThatOneKid/acmcsuf.com/pull/757/commits/54de9a25a956e87b07dfc5346c02dcc6247896f1>.

k4rni reviewed (approved):
These certification pages look extremely cool! It really encourages people to contribute to our website and it's a wonderful way of showing off everyone's contributions. Thank you so much:)

EthanThatOneKid committed:
Delete vite.config.ts.timestamp-1674429041045.mjs

Resolves <https://github.com/EthanThatOneKid/acmcsuf.com/pull/757#discussion_r1083577475>.
